src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -7373,28 +7373,7 @@ static void Main(string[] args)
         }
     }
 }";
-            var expected = @"class Test
-{
-    void method()
-    {
-        static void Main(string[] args)
-        {
-            int v = 0;
-            for(int i=0 ; i<5; i++)
-            {
-                v = NewMethod(v, i);
```

Comment:
@heejaechang If you want flow analysis to give a reason for its failure I think that is a reasonable feature request, but it is a new feature (a new public API) which is currently not available.